File: Java/(Beecrowd)1008 - Salary.java
import java.util.Scanner;
 
public class Main{
 
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        
        int employeeNumber = input.nextInt();
        int workedHours = input.nextInt();
        float receivePerHour = input.nextFloat();
        float salary = workedHours * receivePerHour;
        
        System.out.println("NUMBER = " + employeeNumber);
        System.out.println(String.format("SALARY = U$ %.2f", salary));
    }
 
}

File: C/(Beecrowd)1008 - Salary.c
#include <stdio.h>

int main() {
    int employeNumber, workingHours;
    float receivePerHour, salary;

    scanf("%d", &employeNumber);
    scanf("%d", &workingHours);
    scanf("%f", &receivePerHour);

    salary = workingHours * receivePerHour;

    printf("NUMBER = %d\nSALARY = U$ %.2f\n", employeNumber, salary);

    return 0;
}